Source/Extensions/Blazorise.DataGrid/DataGrid.razor.cs

```diff
@@ -871,11 +933,29 @@ protected IEnumerable<TItem> DisplayData
         /// </summary>
         [Parameter] public TItem SelectedRow { get; set; }
 
+        /// <summary>
+        /// Gets or sets currently selected rows.
+        /// </summary>
+        [Parameter] public List<TItem> SelectedRows { get; set; }
+
+        /// <summary>
+        /// Enables multiple selection
+        /// </summary>
+        [Parameter] public bool MultiSelect { get; set; }

```

Comment:
Might be a good idea to have it as enum `DataGridSelectionMode`

- Single
- Multiple

`public bool SelectionMode { get; set; }`

Reply:
Good Idea. I'll make that change.